dev/dependencyList

```diff
@@ -44,9 +44,10 @@ jackson-dataformat-yaml/2.11.4//jackson-dataformat-yaml-2.11.4.jar
 jackson-datatype-jsr310/2.11.4//jackson-datatype-jsr310-2.11.4.jar
 jackson-jaxrs-base/2.11.4//jackson-jaxrs-base-2.11.4.jar
 jackson-jaxrs-json-provider/2.11.4//jackson-jaxrs-json-provider-2.11.4.jar
-jackson-module-jaxb-annotations/2.9.9//jackson-module-jaxb-annotations-2.9.9.jar
+jackson-module-jaxb-annotations/2.11.4//jackson-module-jaxb-annotations-2.11.4.jar
 jackson-module-paranamer/2.11.4//jackson-module-paranamer-2.11.4.jar
 jackson-module-scala_2.12/2.11.4//jackson-module-scala_2.12-2.11.4.jar
+jakarta.activation-api/1.2.1//jakarta.activation-api-1.2.1.jar
```

Comment:
shall we keep the same version between `jakarta.activation-api` and `jakarta.activation` ?

Reply:
Seems there have different package group, let me investigate